spec/models/service_template_transformation_plan_task_spec.rb

```diff
@@ -459,7 +459,12 @@
 
           it "fails if network IP address is nil" do
             allow(src_nic_2).to receive(:network).and_return(src_network_2b)
```

Comment:
can we rename this `src_network_2b` to be something obvious e.g. `nil_ip_network`?

Reply:
Done. Also renamed `src_network_2a`.